add description about en-gb instead of en-us (#35619)
demisto/content: Packs/rasterize/Integrations/rasterize/README.md

    ```

    To set a language for the browser, add the *--accept-lang* argument followed by the desired language code in IETF BCP 47 format. For example, `--accept-lang=de-DE`.

> If you're willing to set language to en-US please use en-GB instead.

Reviewer: ```suggestion
If you want to set the language to en-US, use en-GB instead.
```
